collective/searchandreplace/searchreplaceutility.py

```diff
@@ -436,9 +446,11 @@ def getRawTextField(obj, field):
             text = safe_unicode(field.getRaw(obj))
     else:
         # Dexterity
-        baseunit = field.get(obj)
+        baseunit = getattr(field.interface(obj), field.__name__)
```

Comment:
Same as above

@mauritsvanrees, does the change from `field.get` to `getattr` misses something ?

Reply:
Seems okay.

>